src/modules/interface/powertoy_module_interface.h

```diff
@@ -28,9 +28,11 @@
     - unload the DLL.
  */
 
+class PowertoySystemMenuIface;
+
 class PowertoyModuleIface {
 public:
-  /* Returns the name of the PowerToy, this will be cached by the runner. */
+  // Returns the name of the PowerToy, this will be cached by the runner.

```

Comment:
I guess I made some confusion with the comments, let's keep how they were.
In general use `//` for comments in the functions body, but it's OK to use `/* */` in the header files of the function description.